client/blocks/nps-survey/docs/example.jsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import NpsSurvey from '../';
+import { isEnabled } from 'config';
+
+class NpsSurveyExample extends PureComponent {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			wasDismissed: false,
```

Comment:
I like to explicitly initialize because I find it makes the code clearer. You can just look at the constructor to see what all the different state props are and what the initial value of them are.